packages/dynamic-import-vars/package-lock.json

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
this file shouldn't exist

Reply:
removed, I probably accidentally installed with npm instead of pnpm or something